src/Render/Concrete.hs

```diff
@@ -254,11 +258,19 @@ instance Render RHS where
 
 instance Render WhereClause where
   render NoWhere = mempty
+#if MIN_VERSION_Agda(2,6,4)
+  render (AnyWhere _range [Module _ _ x [] ds])
+#else
   render (AnyWhere _range [Module _ x [] ds])
+#endif
     | isNoName (unqualify x) =
       vcat ["where", vcat $ fmap render ds]
   render (AnyWhere _range ds) = vcat ["where", vcat $ fmap render ds]
+#if MIN_VERSION_Agda(2,6,4)
+  render (SomeWhere _range _er m a ds) =
```

Comment:
I am currently ignoring erasure annotations but I suppose they _should_ be rendered.